test/bin/build_images.sh

```diff
@@ -478,10 +492,14 @@ configure_package_sources
 
 trap 'osbuild_logs' EXIT
 
-if [ -n "${GROUP}" ]; then
+if [ -n "${LAYER}" ]; then
+    for group in "${TESTDIR}"/image-blueprints/"${LAYER}"/group*; do
+        do_group "${group}" ""
```

Comment:
Does do_group not need to change to understand that the groups are in the layer directory now?

Reply:
Layer is just a logical grouping, so I do not think we need any changes. I have not run the CI yet :). Let me try it just in case.